src/vendor/pg_ruleutils_15.c

```diff
@@ -1,20 +1,18 @@
 /*-------------------------------------------------------------------------
  *
- * ruleutils.c
+ * pg_ruleutils_15.c
  *	  Functions to convert stored expressions/querytrees back to
  *	  source text
  *
- * Portions Copyright (c) 1996-2022, PostgreSQL Global Development Group
+ * Portions Copyright (c) 1996-2023, PostgreSQL Global Development Group
```

Comment:
I think this date change was probably done by accident.

Reply:
> I think this date change was probably done by accident. Can you revert that?

Ok, I'll be more careful next time